Frontend/Admin/src/app/components/list-bikes/list-bikes.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="list">
+    <h3>Rowery</h3>
+    <div class="list-items">
+        <div *ngFor="let bike of bikes"
+            class="list-item list-item-active"
```

Comment:
Zabrałbym klasę list-item-active bo ta lista jest tylko informacyjna i nie ma mieć żadnej fukncjonalności.

Reply:
A nie ma być w tej klasie blokowania rowerów itd?

Reply:
Dodawanie i usuwanie rowerów ma być w widoku stacji. Możemy wrzucić blokowanie w ten widok, ale to w sumie przy innym sprincie.

Reply:
Ok